src/SFA.DAS.EAS.Employer_Account.Database/Scripts/Manual/CON-1712-alterTable.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE [employer_account].[EmployerAgreementTemplate]
```

Comment:
No need for this script. simply open the EmployerAgreementTemplate.sql and add the column